c3.php

```diff
@@ -170,6 +171,14 @@ function __c3_build_crap4j_report(PHP_CodeCoverage $codeCoverage, $path)
         return $path . '.crap4j.xml';
     }
 
+	function __c3_build_cobertura_report(PHP_CodeCoverage $codeCoverage, $path)
+	{
+		$writer = new PHP_CodeCoverage_Report_Cobertura();
```

Comment:
1. Please check if SebastianBergmann\CodeCoverage\Report\Cobertura exists before using it.

2. Your code was indented using tabs, please use 4 spaces instead.

Reply:
I've added a check and fixed indentation.